Automattic/jetpack
# Subscriptions Block: Remove logged in view for wpcom

Fixes https://github.com/Automattic/jetpack/issues/15474
Fixes https://github.com/Automattic/jetpack/issues/18085

#### Changes proposed in this Pull Request:

* Remove the logged in view for WordPress.com so that the Subscriptions block always renders what is in the editor. When a user is logged in to WordPress.com they will still see the email subscription form and can subscribe as normal.

| Before      | After |
| ----------- | ----------- |
| ![](https://user-images.githubusercontent.com/545779/102140643-2cb98580-3e25-11eb-882f-a79e6ebb614a.png) | ![](https://user-images.githubusercontent.com/545779/102140729-507ccb80-3e25-11eb-9b67-23aae3ea6140.png) |

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Apply D56029-code to your sandbox.
* Apply this branch to a local Jetpack install
* Confirm that this works correctly on WordPress.com based on testing instructions on D56029-code
* Confirm the subscriptions form works on your Jetpack site still (you will need it to be a publicly accessible site via jurrassic.tube or ngrok) and the subscriptions Jetpack setting turned on.
* Try subscribing and confirm that you can still subscribe to a site.

#### Proposed changelog entry for your changes:


* Not needed.
